setup.php

```diff
@@ -1323,8 +1331,9 @@ function cloneClicked() {
                                 <input name='port' type='hidden' value='{$installer->port}'>
                                 <input name='loginhost' type='hidden' value='{$installer->loginhost}'>
                                 <input name='dbname' type='hidden' value='{$installer->dbname}'>
-                                <input type='hidden' name='new_theme' id = 'new_theme' value='{$installer->getCurrentTheme()}'>
+                                <input type='hidden' name='new_theme' id = 'new_theme' value=''>
```

Comment:
Note this is reverting a fix for when a user just clicks Keep Current toggle and Proceed to Final Step bug(the theme is then set to blank):
![image](https://user-images.githubusercontent.com/278968/51590309-50c0f600-1e9e-11e9-878b-c58d8d124dbc.png)

Reply:
will fix